_playwright-tests/UI/TemplateCRUD.spec.ts

```diff
@@ -0,0 +1,115 @@
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await page.getByText('Use latest content', { exact: true }).click();
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      // "add template modal"
+      await page.getByText('add template modal', { exact: true });
+      await page.getByPlaceholder('Enter name').fill(`${templateName}`);
+      await page.getByPlaceholder('Description').fill('Template test');
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await page.getByRole('button', { name: 'Create other options' }).click();
+      await page.getByText('Create template only', { exact: true }).click();
+    });
+    await test.step('Read and update values in the template', async () => {
+      await navigateToTemplates(page);
+      await closePopupsIfExist(page);
+      const rowTemplate = await getRowByNameOrUrl(page, templateName);
+      await rowTemplate.getByRole('button', { name: templateName }).click();
+      await expect(page.getByLabel('Breadcrumb').first()).toHaveText('RHELContentTemplates');
+      await page.getByLabel('Breadcrumb').nth(2);
+      await expect(page.getByRole('heading', { level: 1 })).toHaveText(templateName);
+      await expect(page.getByText('Description:Template test')).toBeVisible;
+      await page.getByRole('button', { name: 'Actions' }).click();
+      await page.getByRole('menuitem', { name: 'Edit' }).click();
+      await expect(page.getByText('Define template content')).toBeVisible;
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await expect(page.getByText('Additional Red Hat repositories')).toBeVisible;
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await expect(page.getByText('Custom repositories')).toBeVisible;
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await expect(page.getByText('Set up date')).toBeVisible;
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await expect(page.getByText('Enter template details')).toBeVisible;
+      await expect(page.getByPlaceholder('Enter name')).toHaveValue(`${templateName}`);
+      await expect(page.getByPlaceholder('Description')).toHaveValue('Template test');
+      await page.getByPlaceholder('Enter name').fill(`${templateName}-edited`);
+      await page.getByPlaceholder('Description').fill('Template test edited');
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await page.getByRole('button', { name: 'Confirm changes', exact: true }).click();
+    });
+    await test.step('Delete teh template', async () => {
+      await navigateToTemplates(page);
+      await closePopupsIfExist(page);
+      await page.getByRole('textbox', { name: 'Filter by name' }).fill(`${templateName}-edited`);
+      const rowTemplate = await getRowByNameOrUrl(page, templateName);
```

Comment:
` getRowByNameOrUrl` already changes the filter, so you don't need to set it manually

Reply:
OK, fixed